src/util/cleanup.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Removes all elements on the current page with the given css class.
+ * i.e. jQuery $(`.${className}`).remove();
+ *
+ * @param {...string} classNames - one or more class names to select elements to remove
+ * @returns {void}
+ */
+export const remove = (...classNames) =>
+  [...document.querySelectorAll(classNames.map(className => `.${className}`).join(','))]
+    .forEach(element => element.remove());
```

Comment:
:thinking: we could go even more vanilla

```suggestion
export const removeElementsByClassName = (...classNames) => classNames
    .flatMap(className => [...document.getElementsByClassName(className)])
    .forEach(element => element.remove());
```

Reply:
hmm, actually, `classNames.flatMap(className => [...document.getElementsByClassName(className)])` could be extracted out into its own function (`getElementsByClassNames`?) and then used in both this and the below function.

Reply:
This is true, and indeed feels more elegant and composable! I used to have it as, iirc

```js
export const remove = (...classNames) =>
  classNames.forEach(className =>
    [...document.querySelectorAll(`.${className}`)].forEach(element =>
      element.remove()
    )
  );
```
but combined it into one QSA call for "performance." <- air quotes intended as absolutely giant

Reply:
Well, it didn't clean things up as much as I thought it would, turns out :D